app/forms/dataset_resource_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# Generated via
+#  `rails generate hyrax:work_resource DatasetResource`
+#
+# @see https://github.com/samvera/hyrax/wiki/Hyrax-Valkyrie-Usage-Guide#forms
+# @see https://github.com/samvera/valkyrie/wiki/ChangeSets-and-Dirty-Tracking
+class DatasetResourceForm < Hyrax::Forms::ResourceForm(DatasetResource)
+  include Hyrax::FormFields(:basic_metadata)
+  include Hyrax::FormFields(:dataset_resource)
+  include Hyrax::FormFields(:with_pdf_viewer)
+  include Hyrax::FormFields(:with_video_embed)
+
```

Comment:
Do we need to include [this](https://github.com/samvera/hyku/blob/main/app/forms/image_resource_form.rb#L14)?